spec/lib/rex/proto/kerberos/model/enc_kdc_response_spec.rb

```diff
@@ -225,10 +225,49 @@
                    @tag=27,
                    @tag_class=:UNIVERSAL,
                    @tagging=nil,
-                   @value="DEMO.LOCAL">]>]>]>]>]>]>
+                   @value="DEMO.LOCAL">]>]>]>]>,
+      #<OpenSSL::ASN1::ASN1Data:0x00007ffd80e8cbc0
+       @indefinite_length=false,
+       @tag=12,
+       @tag_class=:CONTEXT_SPECIFIC,
+       @value=
+        [#<OpenSSL::ASN1::Sequence:0x00007ffd80e8cbe8
+          @indefinite_length=false,
+          @tag=16,
+          @tag_class=:UNIVERSAL,
+          @tagging=nil,
+          @value=
+           [#<OpenSSL::ASN1::Sequence:0x00007ffd80e8cc10
+             @indefinite_length=false,
+             @tag=16,
+             @tag_class=:UNIVERSAL,
+             @tagging=nil,
+             @value=
+              [#<OpenSSL::ASN1::ASN1Data:0x00007ffd80e8ccd8
+                @indefinite_length=false,
+                @tag=1,
+                @tag_class=:CONTEXT_SPECIFIC,
+                @value=
+                 [#<OpenSSL::ASN1::Integer:0x00007ffd80e8cd00
+                   @indefinite_length=false,
+                   @tag=2,
+                   @tag_class=:UNIVERSAL,
+                   @tagging=nil,
+                   @value=#<OpenSSL::BN 165>>]>,
+               #<OpenSSL::ASN1::ASN1Data:0x00007ffd80e8cc38
+                @indefinite_length=false,
+                @tag=2,
+                @tag_class=:CONTEXT_SPECIFIC,
+                @value=
+                 [#<OpenSSL::ASN1::OctetString:0x00007ffd80e8cc60
+                   @indefinite_length=false,
+                   @tag=4,
+                   @tag_class=:UNIVERSAL,
+                   @tagging=nil,
+                   @value="\x1F\x00\x00\x00">]>]>]>]>]>]>
 =end
```

Comment:
I'm not sure of the use of this debug output on all of the existing Kerberos tests, but I've kept it for consistency.